radixdlt-core/radixdlt/src/main/java/org/radix/api/jsonrpc/RadixJsonRpcServer.java

```diff
@@ -256,6 +266,22 @@ private JSONObject handle(JSONObject jsonRpcRequest) {
 						result = new JSONObject().put("status", atomStatus.toString());
 					}
 					break;
+				case "Chaos.enableMessageFlooder":
```

Comment:
As mentioned previously, I don't believe it is a good idea to include the sorts of things that have the potential to break the node in the JSON RPC processing pipeline, as it requires stateful stream inspection to firewall them off with nginx.

@rvelaz probably best if you weigh in here from an operational perspective.